scout/server/blueprints/dashboard/controllers.py

```diff
@@ -0,0 +1,254 @@
+        
+    
+    data['cases'] = get_case_groups(adapter, total_cases, institute_id)
+    
+    data['analysis_types'] = get_analysis_types(adapter, total_cases, institute_id)
+
+    # Fetch variant information
+    LOG.info("Fetch sanger variants")
+    validation_query = {'sanger_ordered': True}
+
+    # Case level information
+    validation_cases = set()
+    validated_cases = set()
+    
+    # Variant level information
+    validated_tp = set()
+    validated_fp = set()
+
+    LOG.info("Find all validated variants with query {}".format(institute_id))
+    validation_ordered = adapter.variant_collection.find(validation_query)
+
+    case_ids = general_info['case_ids']
+    nr_ordered = 0
+    for nr_ordered, variant in enumerate(validation_ordered,1):
+        case_id = variant['case_id']
+        if institute_id:
+            if not case_id in case_ids:
+                continue
+        variant_id = variant['_id']
+        # Add case id to validation cases
+        validation_cases.add(case_id)
+
+        validation = variant.get('validation')
+        if validation == 'True positive':
+            validated_tp.add(variant_id)
+            validated_cases.add(case_id)
+
+        elif validation == 'False positive':
+            validated_fp.add(variant_id)
+            validated_cases.add(case_id)
+    
+    nr_validation_cases = len(validation_cases)
+
+    overview = [
+        {
+            'title': 'Phenotype terms',
+            'count': general_info['phenotype_cases'],
+            'percent': general_info['phenotype_cases'] / total_cases,
+        }, 
+        {
+            'title': 'Causative variants',
+            'count': general_info['causative_cases'],
+            'percent': general_info['causative_cases'] / total_cases,
+        }, 
+        {
+            'title': 'Pinned variants',
+            'count': general_info['pinned_cases'],
+            'percent': general_info['pinned_cases'] / total_cases,
+        }, 
+        {
+            'title': 'Cohort tag',
+            'count': general_info['cohort_cases'],
+            'percent': general_info['cohort_cases'] / total_cases,
+        }
+    ]
+    if nr_validation_cases:
+        overview.append(
+        {
+            'title': 'Validation ordered',
+            'count': nr_validation_cases,
+            'percent': nr_validation_cases / total_cases,
+        })
+        overview.append(
+        {
+            'title': 'Validated',
+            'count': len(validated_cases),
+            'percent': len(validated_cases) / nr_validation_cases,
+        })
+
+    data['overview'] = overview
+    
+    variants = []
+    if nr_ordered:
+        variants = [
+            {
+                'title': 'Validation ordered',
+                'count': nr_ordered
+            },
+            {
+                'title': 'Validated True Positive',
+                'count': len(validated_tp),
+                'percent': len(validated_tp) / nr_ordered,
```

Comment:
I still think this should be tp / (tp + fp), in particular as not all clinics (*ahem* cust002 *ahem*) use the verification email extensively yet.

Reply:
Sure I'll fix that